docs/content-server-api/contents/hashId/active-entities/get.md

```diff
@@ -0,0 +1,27 @@
+# Get active entity for a content hash
+
+Get the entity ids of the deployments associated with a content hash
+
+**URL** : `/contents/:hashId/active-entities`
+
+**URL Parameters** : `hashId=[integer]` where `hashId` is the ID of the Hash on the server.
```

Comment:
hashId won't be an integer. It will be a string like `"QmZbAEFPF7LSq9xVZTCAZoHPNV1QX7YiSbKL2g694EA1r7"`